lib/tasks/install-blueprint.js

```diff
@@ -80,6 +113,30 @@ class InstallBlueprintTask extends Task {
     return execa('npm', ['install'], { cwd });
   }
 
+  _npmInstallModule(module, cwd) {
+    logger.info(`Running "npm install ${module}" in "${cwd}" ...`);
+    return execa('npm', ['install', module], { cwd })
+      .then(() => path.join(cwd, 'node_modules', module));
```

Comment:
Should we use `path.dirname(resolve('module/package', { baseDir: cwd })`? I'm unsure there are any cases where what you have now is not correct though, feel free to disregard...